examples/compose/README.txt

```diff
@@ -0,0 +1,6 @@
+.. _compose_examples:
+
+Pipelines and composite estimators
+----------------------------------
+
+Examples of how to compose transformers and pipelines from other estimators. See the :ref:`User Guide <combining estimators>`.
```

Comment:
The link to user guide doesn't work. I guess it might be ``combining_estimators``?